androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/triage/contactreason/RettsClickableItem.kt

```diff
@@ -0,0 +1,51 @@
+package com.EENX15_22_17.digital_journal.android.screens.triage.contactreason
+
+
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.rounded.Close
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.EENX15_22_17.digital_journal.android.ui.theme.colorIcon
+import com.EENX15_22_17.digital_journal.android.ui.theme.colorTextGray
+
+@Composable
```

Comment:
Would be nice to have a preview of this, its a simple component so it should not cause to much trouble